src/txmempool.cpp

```diff
@@ -440,12 +442,27 @@ void CTxMemPool::check(const CCoinsViewCache& active_coins_tip, int64_t spendhei
     // Number of chunks is bounded by number of transactions.
     const auto diagram{GetFeerateDiagram()};
     Assume(diagram.size() <= score_with_topo.size() + 1);
+    Assume(diagram.size() >= 1);
```

Comment:
In commit "Sanity check feerate diagram in CTxMemPool::check()"

In a check function like `CTxMempool::check()`, I think we should always use `assert`/`Assert`, not `Assume`, as we want the checks to always be enabled.